moderators.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Moderators"
+---
+
+A [moderator](governance.md#moderator) is a volunteer gatekeeper
+who manages contributions and discussions from [contributors](governance.md#contributor) and [users](governance.md#user). 
+[Moderators](governance.md#moderator) are vital to the health and success of R-multiverse.
+This page explains the prerequisites, selection criteria, and scope
+of the [moderator](governance.md#moderator) role.
+
+# Prerequisites
+
+[Moderators](governance.md#moderator) must be:
+
+1. Proficient in [R package development](https://r-pkgs.org/).
+1. Comfortable managing content on [GitHub](https://github.com) such as repositories, pull requests, issues, and discussions.
+1. Prepared to facilitate the [process to contribute packages to R-multiverse](https://r-multiverse.org/contributors.html).
```

Comment:
we need some written expectation of responsiveness, as this would help people assess whether they can help. For instance for rOpenSci software review for most things, we sort of expect answers within one week or so.

Reply:
Maybe? My worry is that the docs for moderators have no rough estimate of the workload that the role would entail. Maybe there could be a rough estimate, and also the idea of a trial period after which the moderator can decide it's too time consuming?

Reply:
What if the moderators tell us up front how many hours per week or month they can volunteer, and what they'd be willing to do? Some could stick to reviewing packages (or only certain kinds of packages) while others could just comb the help forums.

I think this right-sized approach can work because there are so many stopping places: there is very little to check when reviewing a package manually, and for a knowledgeable package developer, I think most reviews will end up taking less than 15 minutes. And they can always contact us for help on difficult cases. Feedback on the forums feels similar. By contrast, an rOpenSci package review is a serious investment of time and critical thinking, and it's either all complete or unfinished.

I think the expectation is that any leftover work will fall on the admins.

I have seen this low-pressure leave-anytime model work well for environmental volunteer groups.

Reply:
Would we need to record this information anywhere except the web form responses?